Readme.md

```
@@ -20,15 +20,9 @@ LinkedRecords is configured via environment variables. See table below.
 | PGHOST | localhost | The hostname of the PostgreSQL server. |
 | PGUSER | linkedrecords | The PostgreSQL user name. |
 | PGPASSWORD | xxxx | The PostgreSQL password. |
-| PGDATABASE | xxxx | The PostgreSQL database name. |
-| SERVER_BASE_URL | http://localhost:6543 | The public URL of the linkedrecords server. |
-| FRONTEND_BASE_URL | http://localhost:3001 | The base URL of the frontend. It will be used for the Access-Control-Allow-Origin HTTP header and is also required for the OpenID connect redirection. |
+| PGDATABASE | xxxx | The PostgreSQL database name. ||
```

markdownlint-cli2 (0.17.2)

Table column count
Expected: 3; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)